sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -136,11 +136,13 @@ The following section provides several code snippets using the `client` [created
 Run a Text Analytics predictive model to determine the language that the passed-in document or batch of documents are written in.
 
 ```C# Snippet:DetectLanguage
-string document = "Este documento está en español.";
+            string document = @"Este documento está escrito en un idioma diferente al Inglés. Tiene como objetivo demostrar cómo invocar el método de Detección de idioma del servicio de Text Analytics en Microsoft Azure.
```

Comment:
@pakrym do you have tips on how I can make this formatting better? I want the document to look like a paragraph which is why I am using `@` instead of `+` or `string.Join` but snippets don't render nicely

Reply:
I think it's a bug somewhere in https://github.com/Azure/azure-sdk-for-net/blob/master/eng/SnippetGenerator/DirectoryProcessor.cs#L134 this logic and the `SnippetGenerator` can't detect the amount of indentation to trim.

Reply:
Ah, it's not the bug. Your min indent is 0 because some lines don't have any indentation so snippet generator won't try to re-align them.

Reply:
You can remove the indentation from the original source files or switch to `+` or make SnippetGenerator understand C# and do correct de-indentation :)